versioned_docs/version-5/states.mdx

```diff
@@ -9,19 +9,25 @@ A state describes the machine’s status or mode, which could be as simple as *
 
 These states are “finite”; the machine can only move through the states you’ve pre-defined.
 
-<EmbedMachine name="Video player" embedURL="https://stately.ai/registry/editor/embed/e13bef2b-bb13-4465-96ac-0bc25340688e?machineId=741f69fd-7f01-4932-9407-6871e225bb6d"/>
+<EmbedMachine
+  name="Video player"
+  embedURL="https://stately.ai/registry/editor/embed/e13bef2b-bb13-4465-96ac-0bc25340688e?machineId=741f69fd-7f01-4932-9407-6871e225bb6d"
+/>
 
 :::tip
 
 Watch our [“What are states?” video on YouTube](https://www.youtube.com/watch?v=z-6yhmSWUcc&list=PLvWgkXBB3dd4I_l-djWVU2UGPyBgKfnTQ&index=2) (53s).
 
 :::
 
-<SkipDownLink text="Jump to learning more about the state object in XState" link="#state-object"/>
+<SkipDownLink
+  text="Jump to learning more about the state object in XState"
+  link="#state-object"
+/>
 
 ## Using states in Stately Studio
 
-In Stately Studio, the rounded rectangle boxes are states. The **?** question mark icon in the machine above indicates an unreachable state. The state is unreachable because it isn’t connected to the [initial state](initial-states.mdx) by a [transition](transitions.mdx).
+In Stately Studio, the rounded rectangle boxes are states. The **!** warning icon in the machine above indicates an unreachable state. The state is unreachable because it isn’t connected to the [initial state](initial-states.mdx) by a [transition](transitions.mdx).
```

Comment:
The only substantive change is replacing "? question mark" with "! warning". Everything else seems to be Prettier.

Reply:
@kevinmaes should we take the opportunity to replace this with the Lucide icon?

`<AlertTriangle size={18} />` would do the trick.

Reply:
Sure. Let's do that!